apps/teams-test-app/e2e-test-data/externalAppCardActionsForCEA.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "ExternalAppCardActionsForCEA",
+  "version": ">2.18.0",
```

This version number is possibly not what you want, it will run the e2e tests on versions of TeamsJS that are > 2.18. I think your changes are checked in but not deployed in a TeamsJS release yet, so probably you want to change this to:
```suggestion
  "version": ">=2.29.0",
```

2.28.0 is our most recent release